### PR TITLE
ufw firewall & ssh in iptables

### PR DESCRIPTION
**Pull request by ThirdScript**

### CODE REVIEW

File: iptables [hunk 1]
```diff
@@ -20,6 +20,9 @@ iptables -A INPUT -i eth0 -p tcp --dport 902 -j REJECT --reject-with icmp-port-u
 #	icmp-admin-prohibited
 #	tcp-reset
 
+# To only allow ssh on the server and block all other ports
```
Comment (EmilyGraceSeville7cf) [inline iptables]:
```suggestion
# To only allow ssh on the server and block all other ports:
```

Optional fix: missing `:`.

File: iptables [hunk 2]
```diff
@@ -20,6 +20,9 @@ iptables -A INPUT -i eth0 -p tcp --dport 902 -j REJECT --reject-with icmp-port-u
...
+iptables -A INPUT -p tcp --dport 22 -j ACCEPT
```
Comment (EmilyGraceSeville7cf) [inline iptables]:
```suggestion
iptables --append <INPUT> --protocol tcp --dport 22 --jump <ACCEPT> 
```

Optional fix: missing `<` and `>` around placeholders [capitalized words are recognized as placeholders].
Optional fix: missing long options [man page provides long options].

File: ufw [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# To check the status of the firewall:
+ufw status
+
+# To check the status with more details:
+ufw status verbose
+
+# To allow a port on the firewall by its name:
+ufw allow <port-name>
```
Comment (EmilyGraceSeville7cf) [inline ufw]:
```suggestion
ufw allow <PORT-NAME>
```

Optional fix: missing placeholder capitalization [change placeholder naming style of the first placeholders or change the current placeholder naming style].

**Review by EmilyGraceSeville7cf — APPROVED** [review]
Several optional fixes found.
Thanks for the contribution! 🚀

File: iptables (hunk 2 above)
Comment (ThirdScript) [reply iptables]: Thank you

File: ufw (hunk 3 above)
Comment (ThirdScript) [reply ufw]: Thank you again

**Pull request closed without merging**